Branch-SDK/src/io/branch/indexing/ContentDiscoveryManifest.java

```diff
@@ -36,6 +36,11 @@
     /* Json Array for the content path object and the filtered views for this application */
     private JSONArray contentPaths_;
 
+    /* default scrape repeat delay */
+    private int discoveryRepeatTime_ = 0;
```

Comment:
Content discovery repeat time is per view. Please add  this as part of  path info. Now Content discovery repeat time is added  at manifest level